example/config/README.md

```diff
@@ -0,0 +1,249 @@
+
+- `label_config`:
+    - `query`: The query must return two columns: `entity_id` and `outcome`, based on a given `as_of_date` and `label_timespan`. The `as_of_date` and `label_timespan` must be represented by placeholders marked by curly brackets. The example below reproduces the inspection outcome boolean-or logic: In addition, you can configure what label is given to entities that are in the matrix (see **cohort_config** section) but that do not show up in this label query. By default, these will show up as missing/null.
+    - `include_missing_labels_in_train_as`: However, passing the key `include_missing_labels_in_train_as` allows you to pick True or False.
+    - `name`: In addition to these configuration options, you can pass a name to apply to the label configuration that will be present in matrix metadata for each matrix created by this experiment, under the `label_name` key. The default label_name is `outcome`.
+
+
+### Feature Generation
+The aggregate features to generate for each train/test split. Implemented by wrapping [collate](https://github.com/dssg/triage/tree/config_doc/src/triage/component/collate). Most terminology here is taken directly from [collate](https://github.com/dssg/triage/tree/config_doc/src/triage/component/collate).
+
+Each entry describes a collate.SpacetimeAggregation object, and the arguments needed to create it. Generally, each of these entries controls the features from one source table, though in the case of multiple groups may result in multiple output tables.
+
+Rules specifying how to handle imputation of null values must be explicitly defined in your config file. These can be specified in two places: either within each feature or overall for each type of feature (aggregates_imputation, categoricals_imputation, array_categoricals_imputation). In either case, a rule must be given for each aggregation function (e.g., sum, max, avg, etc) used, or a catch-all can be specified with `all`. Aggregation function-specific rules will take precedence over the `all` rule and feature-specific rules will take precedence over the higher-level rules. Several examples are provided below.
+
+Available Imputation Rules: 
+- `mean`: The average value of the feature (for SpacetimeAggregation the mean is taken within-date).
+- `constant`: Fill with a constant value from a required `value` parameter.
+- `zero`: Fill with zero.
+- `null_category`: Only available for categorical features. Just flag null values with the null category column.
+- `binary_mode`: Only available for aggregate column types. Takes the modal value for a binary feature.
+- `error`: Raise an exception if any null values are encountered for this feature.
+
+    
+- `feature_aggregations`: 
+    - `prefix`: prefix given to the resultant tables
+    - `from_obj`: from_obj is usually a source table but can be an expression, such as a join (ie ```"cool_stuff join other_stuff using (stuff_id)"```)
```

Comment:
do from_objs need to supply their own parens and aliases?

Reply:
actually no